src/main/java/stsjorbsmod/patches/NlothsGiftPatch.java

```diff
@@ -0,0 +1,81 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.shrines.Nloth;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import javassist.CannotCompileException;
+import javassist.CtBehavior;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import stsjorbsmod.relics.FragileMindRelic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class NlothsGiftPatch {
+
+    /**
+     * Removes FragileMindRelic if found in the iterator
+     *
+     * @param iter
+     */
+    public static void removeFragileMindRelic(Iterator<AbstractRelic> iter) {
+        while (iter.hasNext()) {
+            if (FragileMindRelic.ID.equals(iter.next().relicId)) {
+                iter.remove();
+            }
+        }
+    }
+
+    /**
+     * This is called only by edited expressions in the main game. See the following SpirePatch.
+     *
+     * @param playerRelics
+     * @return
+     */
+    public static List<AbstractRelic> clonePlayerRelicsWithoutFragileMind(List<AbstractRelic> playerRelics) {
+        List<AbstractRelic> relics = new ArrayList<>(playerRelics);
+        removeFragileMindRelic(relics.iterator());
+        return relics;
+    }
+
+    @SpirePatch(clz = AbstractDungeon.class, method = "getShrine")
```

Comment:
I think you could actually reuse exactly this same patch on `Nloth.class`/`SpirePatch.CONSTRUCTOR`, if you wanted to gain confidence that the patch implementation worked in the case that's easier to test. You can apply multiple `@SpirePatch` annotations to the same class.

Reply:
(or alternately, have separate @SpirePatch classes but have them both be based on a shared `ExprEditor` class, like how `LegendaryPatch.java` reuses `CloneMasterDeckWithoutLegendearyCardsEditor` for a few different `@SpirePatch`es)

Reply:
Hmm I'll try this.